app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,44 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id, benchmark_type_id, benchmark_result_type_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+    benchmark_type = BenchmarkType.find_by_id(benchmark_type_id)
```

Comment:
`benchmark_run` belongs to `benchmark_type` and `benchmark_result_type`. I don't think you need to be fetching the records with the ids